sgl-kernel/developer_guide.md

```diff
@@ -30,6 +30,15 @@ Steps to add a new kernel:
 4. Expose Python interface in [src/sgl-kernel/\_\_init\_\_.py](https://github.com/sgl-project/sglang/blob/main/sgl-kernel/src/sgl-kernel/__init__.py)
 5. Update [setup.py](https://github.com/sgl-project/sglang/blob/main/sgl-kernel/setup.py) to include new CUDA source
 
+### Clang Format
```

Comment:
remove this

ref https://github.com/sgl-project/sglang/blob/0da0989ad4f468ce35f4c8220241901a75ed1b26/sgl-kernel/Makefile#L24-L25

Reply:
`make format` is enough

Reply:
ok